src/EnergyPlus/HeatBalanceKivaManager.cc

```diff
@@ -695,8 +695,8 @@ bool KivaManager::setupKivaInstances()
 					exposedFraction = expPerimMap[surfNum].exposedFraction;
 				}
 			} else {
-				userSetExposedPerimeter = false;
-				useDetailedExposedPerimeter = true;
```

Comment:
Now it's failing with `exposedFraction` uninitialized when it's used on line 759.